website/static/js/projectCreator.js

```diff
@@ -164,7 +164,7 @@ function ProjectCreatorViewModel(params) {
         return ko.utils.arrayMap(nodes, function(node) {
             return {
                 'id': node.id,
-                'text': node.title
+                'text': $osf.htmlDecode(node.title)
```

Comment:
I'll add a TODO to remove this after we stop storing pre-sanitized strings in the db.